### PR TITLE
Create Signup Form

### PR DESCRIPTION
**Pull request by Bluette1**
# Create Signup Form

- We created a clone of the [mint sign up page](https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26ivid%3D554fdd80-5eb9-4303-ac34-a722de58f740%26adobe_mc%3DMCMID%253D02233895293886739261431122385917781795%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1579687444%26ivid%3D554fdd80-5eb9-4303-ac34-a722de58f740).

### CODE REVIEW

**Comment by shshamim63** [comment]
## Status :red_circle: [!! Changes Required !!] :red_circle: 
___
Hi @Bluette1  and @kelibst :wave: ,
Project looks almost same as the original page. Code looks neat and clean(except two commented code on the css file.).Items are rendering on the place where they should. Awesome job :clap: . But there are some issues that are blocking me from approving this project. Please follow the instructions below to complete the project.
___
### :spades: [Mandatory Changes] :spades: 
___
1. The stickler is not configured yet. But you were suppose to enable it. Please follow the instructions mentioned in this [link](https://github.com/microverseinc/linters-config/tree/master/css) to configure it.
2. PR(Pull Request) Should contain a summary about the task you have covered in here. But I can see any. So please add a summary.
3. You were suppose to check HTML markup with this validator https://validator.w3.org/#validate_by_upload.
I have done so and found 4 errors.
image
4. The type of input field Phone is `tel`.
https://github.com/Bluette1/SignupForm/blob/3672c078161151f67537551195fbc288cf7ffe1f/index.html#L70
5. There are some commented code on the css file, please remove them.
https://github.com/Bluette1/SignupForm/blob/3672c078161151f67537551195fbc288cf7ffe1f/style.css#L42
https://github.com/Bluette1/SignupForm/blob/3672c078161151f67537551195fbc288cf7ffe1f/style.css#L62
___
#### Please fix the above mentioned issue and send another review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/)

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Under Review By Oluwadamilare Tolulope Olusakin(TSE)
Great work team! I am currently writing your review and will be done within the hour, please check on it in a few minutes! :100:

**Review by nikasulo — COMMENTED** [review]
Great work on fixing your HTML validation errors, very nice! Also great that you took into account the comment about the descriptive PR. However, there are a few issues that need to be addressed before you can get approval :+1:, please see my comments below:

File: .stickler.yml
Comment (nikasulo) [inline .stickler.yml]: Please do not ignore files in your stickler config as it is essentially the same as not using stickler at all. Up until recently, I thought stickler was an unnecessary addition to every code base, however, after using it for a while, I have found that it is one of the best tools to ever be created as it has taught me to write more standardized code and also on a more recent project, it saved me hours of time in debugging something I would have never caught with my mere eyes. I hope you come to see the stickler as I have, please give it some strong thought as it is a hard requirement for this project :smile: 
screenshot

File: reset.css
Comment (nikasulo) [inline reset.css]: Consider replacing all of this with the CSS wildcard selector `*`. It means 'everything in the html file' which is essentially what you're trying to reset! :+1:

**Review by nikasulo — CHANGES_REQUESTED** [review]
### Summary
You have done a wonderful job with your project! Please consider the few changes I have proposed, make appropriate changes and fixes for errors that may result when you restore the original stickler configuration and submit for another review!

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

File: reset.css
Comment (kelibst) [reply reset.css]: Thanks a lot for the review

File: .stickler.yml
Comment (Bluette1) [reply .stickler.yml]: Okay, thank you for the review!

**Review by nikasulo — APPROVED** [review]
# Fantastic work team! :rocket: 
Please merge and carry on :handshake: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Comment by Bluette1** [comment]
Thank you very much @Oluwadamilareolusakin!

**Pull request merged**
